Uplift/Views/Cells/FacilitiesDropdownCell.swift

```diff
@@ -72,35 +71,36 @@ class FacilitiesDropdownCell: UICollectionViewCell {
         fatalError("init(coder:) has not been implemented")
     }
 
-    func configure(for facility: Facility,
+    func configure(for facilityDropdown: FacilityDropdown,
                    index: Int,
-                   dropdownStatus: DropdownStatus,
-                   calendarSelectedIndices: [Int: Int],
-                   headerViewTapped: @escaping (Int?, [Int: Int]) -> ()) {
-        self.facility = facility
+                   headerViewTapped: @escaping (Int?) -> Void) {
+        self.facility = facilityDropdown.facility
         self.facilitiesIndex = index
-        self.calendarSelectedIndices = calendarSelectedIndices
         self.headerViewTapped = headerViewTapped
-        dropdownView.updateContentViewHeight(to: FacilitiesDropdownCell.getFacilityHeight(for: facility, calendarSelectedIndices: calendarSelectedIndices) - FacilitiesDropdownCell.headerViewHeight)
+
+        let contentViewHeight = FacilitiesDropdownCell.getFacilityHeight(for: facility) - FacilitiesDropdownCell.headerViewHeight
+        dropdownView.updateContentViewHeight(to: contentViewHeight)
         headerView.configure(for: facility)
+
+        let dropdownStatus = facilityDropdown.dropdownStatus
         if dropdownStatus == .open {
             dropdownView.openDropdown()
             headerView.rotateArrowDown()
         } else {
             dropdownView.closeDropdown()
             headerView.rotateArrowUp()
         }
-        self.dropdownView.layoutIfNeeded()
+        dropdownView.layoutIfNeeded()
         collectionView.reloadData()
     }
 
-    static func getHeight(for facilityDetail: FacilityDetail, index: Int, calendarSelectedIndices: [Int: Int]?) -> CGFloat {
+    static func getHeight(for facilityDetail: FacilityDetail, index: Int) -> CGFloat {
```

Comment:
The `index` argument is no longer needed here

Reply:
good catch